docs/reference_guide.md

```diff
@@ -1193,6 +1215,8 @@ Attaching 1 probe...
 @[32586]: 98
 @[0]: 579
 ```
+Examples in situ:
+[tool: profile_order.bt](https://github.com/iovisor/bpftrace/blob/master/tests/runtime/scripts/profile_order.bt)
```

Comment:
I'd rather not link to runtime tests. We probably won't ever move the `scripts/` dir but I wouldn't bet on runtime test stuff staying still forever. And there's no way anyone will remember to update this if it happens.